pkgs/by-name/op/openvino/package.nix

```diff
@@ -93,6 +93,10 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
+    substituteInPlace src/plugins/intel_cpu/thirdparty/ComputeLibrary/SConstruct \
+      --replace-fail 'toolchain_prefix + "ar"' '"${stdenv.cc.cc}/bin/gcc-ar"' \
+      --replace-fail 'toolchain_prefix + "ranlib"' '"${stdenv.cc.cc}/bin/gcc-ranlib"'
```

Comment:
This will definitely not work with clang 😅 Can we use the generic ar and ranlib instead?

Reply:
No, the GNU toolchain does not install the LTO plugin, so using the prefixed tools is required to understand LTO objects and generate a working static library. See also for example #188544 #384599. A `git grep -w gcc-ar` shows several packages doing the same thing. I suppose the right thing to do is gate the patch behind `stdenv.cc.isGNU`? (LLVM+binutils would need to use `llvm-ar` if that is a supported combination.)

Reply:
@smaeul does this have any consequences for the x86 environments? Seems like this should be rather a fix that needs to happen on openvino side. Is there an Issue thread? Let me know if I'm wrong, thats been my first thoughts ;)